sections/02_intro.md

```diff
@@ -85,10 +81,7 @@ considered infeasible, or if it made difficult, tedious, and non-routine
 analyses routine.
 
 Based on our guiding question, we focused on the application of deep learning to
-topics of biomedical importance. This covers a large range of topics. We divided
-these into three broad classes based on their applied areas. We briefly
-introduce the types of questions, approaches and data which are typical for each
-class in the application of deep learning.
+topics of biomedical importance. We divided the large range of topics into three broad classes based on their applied areas: (1) Disease and Patient Categorization, (2) Fundamental Biological Study, and (3) Patient Treatment. We briefly introduce the types of questions, approaches and data which are typical for each class in the application of deep learning.
```

Comment:
I like listing the topics in text, but think we should drop the numbers since we won't be numbering the section headers.